packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -44,6 +69,13 @@ type Props = {
    * Indicates if ButtonToggle is disabled. Disabled ButtonToggles are inactive and cannot be interacted with. See the [state variant](https://gestalt.pinterest.systems/web/buttontoggle#State) for details on proper usage.
    */
   disabled?: boolean;
+  /**
+   * An icon displayed above the text to illustrate the meaning of the option selected by the ButtonToggle.
+   */
+  graphicIcon?:
```

Comment:
can we do `thumbnail` instead of `graphicIcon` for consistency with Toast

Reply:
We don't actually avoid this first case because this relies on the source image, unfortunately
If the source image is black (or gray, or has gray/black parts to it, or is nearly those colors), I don't think there's a way for us to make sure it can have contrast against dark mode background and disabled grey
I was thinking what the solution for that could be, and between adding our own dynamic color version of that svg into gestalt just for the example, or just making sure the background was always white, I figured that would be the better option
But if you have any other idea, I'd be glad to implement it, it bothers me as well hahah

Reply:
Also I kept the ImageThumbnail wrapper because it handles masking and aria-disabled for the thumbnail itself (it's better that the image be aria-disabled because the Button itself already has accessibility text, or else it would be doubled)

Reply:
DONE, used CSS filters on image tags to ensure icons display properly in dark mode and disabled variants.